### ISSUE TITLE
Log to stdout

### ISSUE THREAD

**Issue by tkellogg** [tkellogg/dura#30]
Last night I made a change to log JSON to `~/.config/dura/logs/`. However, writing to stdout has a lot of advantages when running with other tools, like systemd.

- Print to stdout by default
- Print to file if `--log-file` is specified

**Comment by Nymouz7** [comment]
Wow wow

**Comment by plaxi0s** [comment]
Does it make sense to use a pre-existing logger like [slog](https://github.com/slog-rs/slog)?

**Comment by tkellogg** [comment]
@plaxi0s besides being an amazing name for a logging library, it does suggest `tracing` instead. Do you have a feel for which is better? I think i'd rather the newer option, but i don't have any strong opinions.

**Comment by plaxi0s** [comment]
yeah I agree, we should definitely use the `tracing` crate. I'll try to work on this one.

**Comment by plaxi0s** [comment]
To enable logging to a log file we need to take an optional filepath in `dura serve`.
Both `dura serve --logfile <path_to_logfile>` and `dura serve <path_to_logfile>` should work.

**Comment by tkellogg** [comment]
I prefer the option approach
